internal/ethapi/api.go

```diff
@@ -773,6 +781,27 @@ func (s *PublicBlockChainAPI) Health() bool {
 	return true
 }
 
+// GetFinalizedBlock returns the requested finalized block.
+// * probabilisticFinalized should be in range [2,21], then the block with number `max(voteFinalized, latest-probabilisticFinalized)` is returned
+func (s *PublicBlockChainAPI) GetFinalizedBlock(ctx context.Context, probabilisticFinalized int64, fullTx bool) (map[string]interface{}, error) {
```

Comment:
what does fulltx mean?

Reply:
add in comment
When fullTx is true all transactions in the block are returned, otherwise  only the transaction hash is returned.